src/components/DateTemperature/index1.js

```diff
@@ -0,0 +1,45 @@
+export const MainBlockDay = ({ data, i, j, title }) => {
```

Comment:
what does mean i and j props? call the variables more describe

Reply:
Is that real MainBlockDay? In my opinion the component should call as IntervalTemperature